Softeq.XToolkit.Permissions.Droid/Softeq.XToolkit.Permissions.Droid.csproj

```diff
@@ -26,6 +25,8 @@
     <AndroidLinkMode>None</AndroidLinkMode>
     <JavaMaximumHeapSize>
     </JavaMaximumHeapSize>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>

```

Comment:
Please set lang version and `<Nullable>` in the shared `<PropertyGroup>`, instead of build configurations (VS default behavior)